src/main.cpp

```diff
@@ -481,6 +481,74 @@ static wf_recorder_output* detect_output_from_region(const capture_region& regio
     return nullptr;
 }
 
+static void help(){
+
+    printf("\
```

Comment:
You can use C++11 raw strings here. Just do

```
printf(R"(
blah
blah
)");

```

In this case, you won't have to add `\n\` to every line.

Reply:
Will change it .  Didn't knew about this improvement. Also was took too much time doing the `\n\ ` way...